inventory/hosts.ini

```
@@ -19,3 +19,5 @@ cluster_dns=192.168.1.1 9.9.9.9
 
 k3s_username='k3s'
 k3s_version='v0.10.1'
+
+ssh_key=~/.ssh/ssh_slm@bub.lan_id_rsa
```

Hi @slmingol,
I like your ideas and going to merge them. How about renaming [hosts.ini](https://github.com/OmegaSquad82/ansible-k3sup/blob/master/inventory/hosts.ini ) to _example-hosts-ini_ so users are being forced to create their own version? I've already added *hosts.ini* to [.gitignore](https://github.com/OmegaSquad82/ansible-k3sup/blob/master/.gitignore )
```suggestion
ssh_key=~/.ssh/id_rsa
```